stdlib/genericpath.pyi

```diff
@@ -1,17 +1,14 @@
 import sys
-from _typeshed import AnyPath
-from os.path import commonprefix as commonprefix
-from typing import Text
-
-def exists(path: AnyPath) -> bool: ...
-def isfile(path: Text) -> bool: ...
-def isdir(s: Text) -> bool: ...
-def getsize(filename: Text) -> int: ...
-def getmtime(filename: Text) -> float: ...
-def getatime(filename: Text) -> float: ...
-def getctime(filename: Text) -> float: ...
+from os.path import (
+    commonprefix as commonprefix,
+    exists as exists,
+    getatime as getatime,
+    getctime as getctime,
+    getmtime as getmtime,
+    getsize as getsize,
+    isdir as isdir,
+    isfile as isfile,
+)
```

Comment:
It might make more sense if `os.path` imports from `genericpath`. That's what actually happens at runtime.

Reply:
I think putting `from posixpath import foo as foo` into `ntpath` along with a comment would be the best.